packages/storybook-react/src/stories/LoginPage/LoginPage.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import {
+  ButtonLink,
+  Heading1,
+  Heading2,
+  Link,
+  Page,
+  PageContent,
+  PageFooter,
+  PageHeader,
+  Paragraph,
+  Separator,
+} from '@utrecht/component-library-react';
+import {
+  UtrechtBreadcrumb,
+  UtrechtDigidButton,
+  UtrechtIconArrow,
+  UtrechtLogo,
+} from '@utrecht/web-component-library-react';
+import React from 'react';
+
+export default {
+  title: 'React.js Component/Login page',
+  id: 'react-login-page',
+  component: Page,
+} as ComponentMeta<typeof Page>;
+
+const Template: ComponentStory<typeof Page> = (args) => (
+  <Page {...args}>
```

Comment:
Not sure how helpful `{...args}` on `Page` is, we probably might as well leave it out.